docs/reference/io.rst

```diff
@@ -1,48 +1,56 @@
 Input/output (`sunpy.io`)
 *************************
 
+The primary focus of ``sunpy.io`` is to provide a common interface for reading and writing.
 ``sunpy.io`` contains readers for files that are commonly used in solar physics.
 
-Unified File Readers
-====================
+These include:
 
-.. automodapi:: sunpy.io
+- Public API
+  - GENX
+  - ANA
+  - NOAA SWPC Solar Region Summary (SRS)
+  - ASDF
 
-.. automodapi:: sunpy.io.ana
+- Internal/Private API
+  - FITS
+  - JPEG2000
+  - CDF
```

Comment:
I understand the point here but if we want this to be  private API, why mention it at all?

Reply:
Removed.